.babelrc.js

```diff
@@ -32,11 +43,4 @@ if (process.env.BABEL_ENV === "cov") {
   config.plugins.push("babel-plugin-istanbul");
```

Comment:
should this still be here? Jest adds it itself (and you removed it from package.json)

Reply:
good catch